src/orchestrator.rs

```diff
@@ -81,6 +82,10 @@ impl Orchestrator {
         Ok(orchestrator)
     }
 
+    pub fn get_config(&self) -> OrchestratorConfig {
```

Comment:
iirc rust typically uses a `foo() / set_foo()` rather than `get_foo() / set_foo()` convention for getters/setters? At least thats what typically seen in `std`
nit/suggestion: `get_config()` -> just `config()`